config/locales/en/citizens.yml

```diff
@@ -59,6 +59,7 @@ en:
         title: Do you agree to share your bank account information with the LAA?
         inset_text: We only use your financial information to support your legal aid application. We do not use it for anything else.
         summary_heading: What information will the LAA be able to see?
+        true_layer_auth_error: Select yes if you agree to share your bank account information with the LAA
```

Comment:
Is it worth getting Jim to review the message?  If they have selected `yes` and authentication fails it could be very confusing for citizen users?